docs/README.md

```diff
@@ -1,5 +1 @@
-# EICrecon
-
-**JANA based reconstruction for the EPIC detector**
-
-<img src="demo.png" style="max-height: 300px"/>
+Replaced
```

Comment:
Please revert, this was merged accidentally

Reply:
```suggestion
# EICrecon

**JANA based reconstruction for the EPIC detector**

<img src="demo.png" style="max-height: 300px"/>
```

Reply:
Reverted.